pkg/controller/podautoscaler/OWNERS

```diff
@@ -1,14 +1,17 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
+  - adrianmoisey
   - gjtempleton
-  - mwielgus
+  - omerap12
+  - raywainman
 approvers:
-  - mwielgus
+  - gjtempleton
```

Comment:
I talked with Guy about it, he's not there yet [meeting the approver requirements](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-2), so this can't be modified yet.